opencensus/trace/propagation/text_format.py

```diff
@@ -49,7 +49,7 @@ def from_carrier(self, carrier):
             if key == _SPAN_ID_KEY:
                 span_id = carrier[key]
             if key == _TRACE_OPTIONS_KEY:
-                trace_options = bool(carrier[key])
+                trace_options = carrier[key]
```

Comment:
Not directly related to this PR. It seems we don't have any validation here. @c24t for awareness, we need to have a better implementation going forward.